src/modules/auto-oembed.js

```diff
@@ -40,8 +40,12 @@
 	var embedOne = function (ctx, force) {
 		var vPlayer = ctx.find(PLAYER_SEL);
 		var autoLoad = vPlayer.attr('data-autoload');
-		
-		if (!force) {
+
+		var provider = vPlayer.attr('data-oembed-provider');
+		var videoPlayers = ['Vimeo', 'Youtube'];
+		var isVideo = provider &&  ( 0 > $.inArray( provider, videoPlayers ) );
+
+		if (!(force || !isVideo)) {
```

Comment:
Would be better to eliminate the !()

```suggestion
		if (!force && isVideo) {
```